run-service.sh

```diff
@@ -18,8 +18,8 @@
 
 ARGS=$@
 APPHOME=home
-. apphome.sh
 
+MAVEN_OPTS="-agentlib:jdwp=transport=dt_socket,server=y,address=8000,suspend=n" \
```

Comment:
I am not so sure that it is a good idea to add the explanation to every project. I think this should become basic knowlegde for every developer. Basically `MAVEN_OPTS` is the environment variable that Maven uses to add jvm options to its invocation. The Options used here cause the jvm to run as a debug server. The can all be found in the `java` executable help:

From `man java`:
```
-agentlib:libname[=options]
                           Load native agent library libname, e.g.,

        -agentlib:hprof
        -agentlib:jdwp=help
        -agentlib:hprof=help

For more information, see JVMTI Agent Command Line Options at http://java.sun.com/j2se/1.5.0/docs/guide/jvmti/jvmti.html
```

The running `java -agentlib:jdwp=help`:
```
               Java Debugger JDWP Agent Library
               --------------------------------

  (see http://java.sun.com/products/jpda for more information)

jdwp usage: java -agentlib:jdwp=[help]|[<option>=<value>, ...]

Option Name and Value            Description                       Default
---------------------            -----------                       -------
suspend=y|n                      wait on startup?                  y
transport=<name>                 transport spec                    none
address=<listen/attach address>  transport spec                    ""
server=y|n                       listen for debugger?              n
launch=<command line>            run debugger on event             none
onthrow=<exception name>         debug on throw                    none
onuncaught=y|n                   debug on any uncaught?            n
timeout=<timeout value>          for listen/attach in milliseconds n
mutf8=y|n                        output modified utf-8             n
quiet=y|n                        control over terminal messages    n

Obsolete Options
----------------
strict=y|n
stdalloc=y|n

Examples
--------
  - Using sockets connect to a debugger at a specific address:
    java -agentlib:jdwp=transport=dt_socket,address=localhost:8000 ...
  - Using sockets listen for a debugger to attach:
    java -agentlib:jdwp=transport=dt_socket,server=y,suspend=y ...

Notes
-----
  - A timeout value of 0 (the default) is no timeout.

Warnings
--------
  - The older -Xrunjdwp interface can still be used, but will be removed in
    a future release, for example:
        java -Xdebug -Xrunjdwp:[help]|[<option>=<value>, ...]

```